README.md

```diff
@@ -1,8 +1,10 @@
 EnsEMBL - Production Reporting Applications
 ===========================================
 
-This repository contains Python applications used for reporting status and
-outcomes of EnsEMBL Production Pipelines and other internal Production services.
+Python applications used for reporting status and outcomes of EnsEMBL Production
+Pipelines and other internal Production services.
+
+[![Build Status](https://travis-ci.com/Ensembl/ensembl-prodinf-reporting.svg?token=5zKzvNKrmopKSdQGqBxH&branch=main)](https://travis-ci.com/Ensembl/ensembl-prodinf-reporting) [![License](https://img.shields.io/badge/license-Apache--2.0-blue)](https://github.com/Ensembl/ensembl-prodinf-reporting/blob/main/LICENSE)
```

Comment:
Shouldn't it be at the tom of README file? (just asking)

Reply:
I don't think there's a specific place were to put badges. IMO they look nicer right after project title and short description, or after the title if the description is longer than few lines.

Reply:
I feel like it's better to have all of them on top of the title, personally. :-)